cookbooks/aws-parallelcluster-platform/templates/base/parallelcluster_supervisord.conf.erb

```diff
@@ -39,7 +39,15 @@ command = <%= node['cluster']['dcv']['authenticator']['virtualenv_path'] %>/bin/
 user = <%= node['cluster']['dcv']['authenticator']['user'] %>
 environment = HOME="<%= node['cluster']['dcv']['authenticator']['user_home'] %>",USER="<%= node['cluster']['dcv']['authenticator']['user'] %>"<% if @region.start_with?('us-iso') -%>,AWS_CA_BUNDLE="<%= @aws_ca_bundle %>"<% end -%>
 <% end -%>
-
+<% if node['cluster']['scheduler'] == 'slurm' && node['cluster']['job_info_enabled'] -%>
```

Comment:
Are we going to pass this setting as part of the dna.json or are we going to get it from the cluster config yaml? I ask because, if we get it from the yaml, then I think you might need to use a lazy evaluator to correctly resolve the value.